src/debugger.ts

```diff
@@ -253,7 +277,7 @@ export class Debugger
       // If the Ruby debug exits with an exit code > 1, then an error might've occurred. The reason we don't use only
       // code zero here is because debug actually exits with 1 if the user cancels the debug session, which is not
       // actually an error
-      this.debugProcess.on("exit", (code) => {
+      this.debugProcess.on("close", (code) => {
```

Comment:
The `exit` event actually happens before the stdin, stdout and stderr pipes are closed. If we raise an error on `exit`, we never capture the backtrace of the server printed to `stderr`.

But if we use the `close` event, then everything gets printed and it's significantly easier to understand why the debug server failed to start.